### PR TITLE
Removes presentational styles from container utility

### PR DESCRIPTION
**Pull request by sky-jack**
This will be a breaking change for users of Springer Nature brand context that rely on the container utility for card/container styles.

### CODE REVIEW

File: context/brand-context/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@springernature/brand-context",
-  "version": "10.1.1",
+  "version": "10.1.2",
```
Comment (tulipdexter) [inline context/brand-context/package.json:3]: Looks like you've bumped a patch but meant a major

**Review by tulipdexter — CHANGES_REQUESTED** [review]

File: context/brand-context/package.json (hunk 1 above)
Comment (sky-jack) [reply context/brand-context/package.json:3]: Sorry. I should have removed that text from the PR. I discussed this with Alex and he suggested it should be a patch.

Comment (tulipdexter) [reply context/brand-context/package.json:3]: Ah I see. In the case the `History` entry is incorrect 😄

File: context/brand-context/HISTORY.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 # History
 
+## 11.0.0 (2021-02-25)
```
Comment (tulipdexter) [inline context/brand-context/HISTORY.md:3]:
```suggestion
## 10.1.2 (2021-02-25)
```

**Review by tulipdexter — CHANGES_REQUESTED** [review]

**Review by tulipdexter — APPROVED** [review]
Nice one @sky-jack!

**Comment by alexkilgour** [comment]
This change was already approved but not merge. Due to another incoming change, this meant that the version was out of date. I have bumped the version and merged this PR to avoid this happening again.

**Pull request merged**